sdk/storage/storage-blob/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@azure/storage-blob",
   "sdk-type": "client",
-  "version": "12.20.0",
+  "version": "12.23.0-beta.1",
```

Comment:
Just a little curious.
12.20.0->12.23.0, for the version between them, will it be dedicated to the V1 SDK release before CoreV2 GA?
Will it cause we first release 12.23.0, then 12.21.0? Will customer be confused?